Pset_Maintenance and Pset_Purchase (#341)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/r/RepairingContent_2uB5y1E3H1zuDscjqI6Rjw/Documentation.md

> Content of repair, reason and nature can be given, e.g. display faults, communication failure, display exchange.

Reviewer: ```suggestion
Content of the property involves repair, reason and nature, e.g. display faults, communication failure, display exchange.
```

> Content of repair, reason and nature can be given, e.g. display faults, communication failure, display exchange.

Reviewer 2: RepairContent
